src/entities/assetTypes.ts

```diff
@@ -1,17 +1,23 @@
 export enum AssetType {
+  arbitrum = 'arbitrum',
```

Comment:
No changes required - just a note thinking about asset type definition (what about polygon nfts?) do we want to keep network types as a separate parameter on the asset?

Reply:
I think we do, it makes it easier to have a polygonNFT type to deal with the send logic.